Makefile

```diff
@@ -75,9 +75,12 @@ build-unit-tests-image:
 run-unit-tests:
 	$(run_unit_tests_image) make test
 
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
```

Comment:
I thought we [agreed](https://github.com/IBM/ibm-block-csi-operator/pull/216#r728721064) not to use pwd

Reply:
not relevant anymore